dev/tests/integration/testsuite/Magento/Quote/Model/QuoteTest.php

```diff
@@ -393,7 +393,7 @@ protected function _getCustomerDataArray()
             \Magento\Customer\Model\Data\Customer::DOB => '2014-02-03 00:00:00',
             \Magento\Customer\Model\Data\Customer::EMAIL => 'qa@example.com',
             \Magento\Customer\Model\Data\Customer::FIRSTNAME => 'Joe',
-            \Magento\Customer\Model\Data\Customer::GENDER => 'Male',
+            \Magento\Customer\Model\Data\Customer::GENDER => 0,
```

Comment:
Why need this change?

Reply:
Fixed fixture implementation according to interface doc block annotation https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Customer/Api/Data/CustomerInterface.php#L269. After replacing `get_class` by namespace this fixture breaks test.